resources/js/Events/Components/NodeLabel.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useRef, useState } from "react";
+import React from "react";
+import Checkbox from "./Checkbox.jsx";
+import { EnabledTriangle, DisabledTriangle } from "./Icons.jsx";
+
+const fixTitle = (title) => {
+  title = title.replace(/u03b1/g, "α");
+  title = title.replace(/u03b2/g, "β");
+  title = title.replace(/u03b3/g, "γ");
+  title = title.replace(/u00b1/g, "±");
+  title = title.replace(/u00b2/g, "²");
+
+  return title;
+};
```

Comment:
A more general solution to work for all unicode characters:
```suggestion
const fixTitle = (title) => {
  return title.replace(/\\u([\da-fA-F]{4})/g, function (m, $1) {
    return String.fromCharCode(parseInt($1, 16));
  });
};
```